libphal/log.H

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
I don't know what `pragma once` does.  But each header should have #ifndef to avoid multiple inclusions of the header file.